### PR TITLE
Split PostScraper into PostScraper and ReplyScraper

### PR DESCRIPTION
**Pull request by teceler**
(split from #1224)

Codeclimate has complaints because I tried to change things as minimally as possible in the move.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -200,7 +200,7 @@ Rails/ReadWriteAttribute:
 # post_scraper.rb is run manually in the command line so stdout is fine
```
Comment (Marri) [inline .rubocop.yml:200]: also this

File: app/services/post_scraper.rb [hunk 2]
```diff
@@ -1,25 +1,5 @@
 class PostScraper < Object
   SANDBOX_ID = Rails.env.production? ? 3 : 5
```
Comment (Marri) [inline app/services/post_scraper.rb:2]: is this even still correct?

Comment (teceler) [reply app/services/post_scraper.rb:2]: It is not!

File: .rubocop.yml (hunk 1 above)
Comment (teceler) [reply .rubocop.yml:200]: PostScraper is in fact the part that's run so it seemed more correct to leave that?

Comment (Marri) [reply .rubocop.yml:200]: Valid, could be ~any more clear that post_scraper calls reply_scraper but it's probably fine as is

**Review by Marri — APPROVED** [review]

**Pull request merged**